lib/domain/entities/todo_entity.dart

```diff
@@ -1,3 +1,16 @@
 class TodoEntity {
-  TodoEntity();
+  String id;
+  String owner;
+  bool completed;
+  String description;
+  DateTime createdAt;
```

Comment:
created and updated doesnt have significance in presentation layer. hence we can remove them as well